### PR TITLE
HW2 - afs2185

### PR DESCRIPTION
**Pull request by afsong**
add multithreading library description

Add serenity description

Add link to markdown files

fix path

add hw2 link

### CODE REVIEW

File: students/afs2185.md [hunk 1]
```diff
@@ -4,22 +4,15 @@ Name: Annie Song
...
-I'd like to work on one of the following possible projects:
+GitHub README: [link](https://github.com/afsong/afsong/blob/master/README.md)
 
-- Network Library in C++
-  - Considering how there is no networking code available in the standard library, I want to create a simple library that can handle basic network functionalities and protocols (e.g. DNS, HTTP 1.0, etc).
-- Multithreading Library in C++
-  - I want to create a general purpose library that provides a userful multithreading interface such as thread pools and job queues.
-    `
+## Project Proposal
+[MTLibrary](./projects/cpp/multithreading.md)
```
Comment (timkpaine) [inline students/afs2185.md:12]: update link path and re-squash 😈

File: students/afs2185.md [hunk 2]
```diff
@@ -4,22 +4,15 @@ Name: Annie Song
...
 
 ## Project Contributions
+[SerenityOS](./projects/cpp/SerenityOS.md)
```
Comment (timkpaine) [inline students/afs2185.md:15]: update link path and re-squash 😈

File: projects/cpp/multithreading.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Multithreading Library in C++
```
Comment (timkpaine) [inline projects/cpp/multithreading.md:1]: add project link here

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/cpp/multithreading.md (hunk 3 above)
Comment (afsong) [reply projects/cpp/multithreading.md:1]: added in the correct file

File: students/afs2185.md (hunk 2 above)
Comment (afsong) [reply students/afs2185.md]: link path updated

File: students/afs2185.md (hunk 1 above)
Comment (afsong) [reply students/afs2185.md]: link path updated

**Pull request merged**